src/store/ChartTitleStore.js

```diff
@@ -51,6 +51,7 @@ export default class ChartTitleStore {
     @observable todayChange = null;
     @observable todayChangePercent = null;
     @observable isVisible = false;
+    @observable enableShowPrice = false;
```

Comment:
making `enableShowPrice` as observable is not required here. There is no reaction depending on this. I think we can just use it as normal variable

Reply:
yes, you are right. nice catch, just updated it